unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Shaders/UnlitCutoutTinted.shader

```diff
@@ -50,7 +52,11 @@ SubShader {
  
             fixed4 frag (v2f i) : SV_Target
             {
-                fixed4 col = tex2D(_BaseMap, i.texcoord) * _BaseColor;
+                fixed4 col = tex2D(_BaseMap, i.texcoord);
+                fixed4 tintMask = tex2D(_TintMask, i.texcoord);
+
+                col *= lerp(float4(1,1,1,1), _BaseColor, tintMask.r);
```

Comment:
In `EyesShader.shader` I see that the lerp function does `( 1.0 - tex2D( _IrisMask, uv_IrisMask ).r )` I think we should invert this one too so the pipeline keeps coherency.